source/mesh_system.c

```diff
@@ -62,6 +70,7 @@ void mesh_system_init(void)
         platform_timer_enable();
         eventOS_scheduler_init();
         trace_init(); // trace system needs to be initialized right after eventOS_scheduler_init
+        set_trace_print_function(trace_print);
```

Comment:
It might cause "unexpected" results if application overwrite this after mesh_system_init(), but probably this is only easy  way to fix default line feeds.. I'm fine with this. Just thinking that should this be in trace lib instead of..